XamlControlsGallery/ControlPages/NavigationViewPage.xaml

```diff
@@ -375,5 +379,56 @@
                 <local:ControlExampleSubstitution x:Name="navViewASB" Key="NavViewASB"/>
             </local:ControlExample.Substitutions>
         </local:ControlExample>
+
+        <local:ControlExample Grid.Row="10" 
+                              HeaderText="Hierarchical NavigationView"
+                              VerticalAlignment="Top" 
+                              HorizontalContentAlignment="Stretch" 
+                              XamlSource="NavigationView\NavigationViewSample8_xaml.txt"
+                              WebViewHeight="400">
+            <local:ControlExample.Options>
+                <StackPanel>
+                    <RadioButton x:Name="nvSample8Left" Content="Left mode" Checked="panePositionLeft_Checked" IsChecked="True"/>
+                    <RadioButton x:Name="nvSample8Top" Content="Top mode" Checked="panePositionTop_Checked"/>
+                    <RadioButton x:Name="nvSample8LeftCompact" Content="LeftCompact mode" Checked="panePositionLeftCompact_Checked"/>
+                </StackPanel>
+            </local:ControlExample.Options>
+            <Grid>
+                <Grid.RowDefinitions>
+                    <RowDefinition Height="Auto"/>
+                    <RowDefinition Height="Auto"/>
+                </Grid.RowDefinitions>
+
+                <TextBlock TextWrapping="WrapWholeWords"
+                           Margin="0,0,0,12"
+                           Grid.Row="0"
+                           Text="NavigationView supports hierarchy in Left, LeftCompact, and Top display modes. Note that Left or Top mode works best if your parent items should navigate to specific pages, rather than just serve as headings. To expand the sub-menu without selecting the parent item, click the chevron. Choose between the three modes to see how each works." />
+
+                <muxc:NavigationView x:Name="nvSample8" Grid.Row="1" Height="460"
+                                        PaneDisplayMode="Left" 
+                                        IsTabStop="False" 
+                                        SelectionChanged="NavigationView_SelectionChanged8">
+                    <muxc:NavigationView.MenuItems>
+                        <muxc:NavigationViewItem Content="Home" Icon="Home" ToolTipService.ToolTip="Home" Tag="SamplePage1"/>
+                        <muxc:NavigationViewItem Content="Parent Item 1" Icon="Save" ToolTipService.ToolTip="Parent Item 1" Tag="SamplePage2">
```

Comment:
I know the icons are mostly just placeholders, but are there other icons that help emphasize the hierarchical layout? Maybe like an icon that is a group of people as the root, and then each sub-item is just a single person? Not all the icons need to be different.

Reply:
Changed it to some icons that have a natural hierarchy and make more sense.